apps/dapp/src/components/Pages/Ascend/components/LBPForm/index.tsx

```diff
@@ -119,73 +86,133 @@ export const LBPForm = ({ pool }: Props) => {
+    return updateWeightsGradually.handler(
+      formValues.startDate,
+      formValues.endDate,
+      formValues.endWeight1,
+      formValues.endWeight2
+    );
   };
 
-  const pausePool = () => {
-    console.log('Invoke pause pool');
+  const updateSwapEnabled = async (enabled: boolean) => {
+    return setSwapEnabled.handler(enabled);
   };
 
   const drainPool = () => {
     console.log('Invoke drain pool');
   };
 
-  // TODO: Do we need any of the other fields in the form? I suspect we do.
+  const formatDate = (date: Date) => {
+    if (!date) return;
+    let result;
+    try {
+      result = format(date, "yyyy-LL-dd'T'kk:mm");
+      return result;
+    } catch (error) {
+      console.error(error);
+      result = format(new Date(), "yyyy-LL-dd'T'kk:mm");
+    }
+    return result;
+  };
+
+  const formatWeight = (weight: DecimalBigNumber) => {
+    if (!weight) return '0';
+    return Math.trunc(Number(formatUnits(weight.value, 16)));
+  };
 
   return (
     <Form>
-      <h2>{formValues.id ? 'Edit' : 'Create'} LBP</h2>
+      <h2>{isEditMode ? 'Edit' : 'Create'} LBP</h2>
+      <Link to="/dapp/ascend/admin">Back to List</Link>
+      <br />
       <InputGroup hidden={!formValues.id}>
         <Label htmlFor="address">Address</Label>
         <Input type="text" id="name" disabled readOnly value={formValues.address} />
       </InputGroup>
       <InputGroup>
         <Label htmlFor="name">Name</Label>
-        <Input type="text" id="name" required onChange={createChangeHandler('name')} value={formValues.name} />
+        <Input
+          type="text"
+          id="name"
+          required
+          disabled={isEditMode}
+          readOnly={isEditMode}
+          onChange={createChangeHandler('name')}
+          value={formValues.name}
+        />
       </InputGroup>
       <InputGroup>
-        <Label htmlFor="overview">Overview</Label>
-        <TextArea required id="overview" onChange={createChangeHandler('overview')} value={formValues.overview} />
+        <Label htmlFor="symbol">Symbol</Label>
+        <Input
+          type="text"
+          id="symbol"
+          required
+          disabled={isEditMode}
+          readOnly={isEditMode}
+          onChange={createChangeHandler('symbol')}
+          value={formValues.symbol}
+        />
       </InputGroup>
       <InputGroup>
         <Label htmlFor="released-token">Released</Label>
-        <Select id="released-token" onChange={createChangeHandler('releasedToken')} value={formValues.releasedToken}>
-          {Object.values(env.tokens).map((token) => (
-            <option value={token.address} key={token.address}>
-              {token.name}
-            </option>
-          ))}
-        </Select>
+        {isEditMode ? (
+          <h3>{pool.tokens[0].symbol}</h3>
+        ) : (
+          <Select id="released-token" onChange={createChangeHandler('releasedToken')} value={formValues.releasedToken}>
+            {Object.values(env.tokens).map((token) => (
```

Comment:
we'll need to clean up the `env.tokens` as a follow up here.  We should probably validate what tokens we care about here and make sure the env values are correct

Reply:
doesnt need to be part of this PR

Reply:
Yes was thinking that could be a follow up PR as well. I will clarify in the channel and get that going.